Framework/Geometry/inc/MantidGeometry/Instrument/ComponentInfoBankHelpers.h

```diff
@@ -2,6 +2,7 @@
 #define MANTID_GEOMETRY_COMPONENTINFOBANKHELPERS_H_
 
 #include "MantidGeometry/DllConfig.h"
+#include "MantidKernel/EigenConversionHelpers.h"
```

Comment:
Don't include this, but do include eigen header

Reply:
done